src/core/unit.c

```diff
@@ -1067,6 +1067,15 @@ const char *unit_description(Unit *u) {
         return strna(u->id);
 }
 
+const char *unit_identifier(Unit *u) {
```

Comment:
hmm, more name nitpicking: I find this function name quite confusing, i.e. that unit_identifier() is something different from unit->id is a bit weird... i.e. id and identifier is synonymous at least in my head, and it is too in this fucntion sometimes, but also usually not

Maybe call it "unit_string()"? or "unit_status_string()"? (i prefer the latter)